contrib/de_DE/LC_MESSAGES/hugin.po

```diff
@@ -0,0 +1,42 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-01-31 22:59+0100\n"
+"PO-Revision-Date: 2016-01-31 23:09+0100\n"
+"Language: de_DE\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: Tobias Jakobs <tobias.jakobs@gmail.com>\n"
+"Language-Team: \n"
+"X-Generator: Poedit 1.8.4\n"
+
+#: hugin.lua:73
+msgid "hugin not found"
+msgstr "Hugin wurde nicht gefunden"
+
+#: hugin.lua:97
+msgid "Will try to stitch now"
+msgstr "Versuche Bilder zusammen zu fügen"
```

Comment:
should be "zusammenzufügen", see http://www.duden.de/rechtschreibung/zusammenfuegen